hedera-node/docs/system-accounts-operations.md

```diff
@@ -107,7 +107,9 @@ Legend of the symbols used in the tables below:
 | 0x3E8 → ♾️                           |                     ✅                      | ✅ - if account exists and has receiverSigRequired = false<br/>✅ - if (account exists) and (account has receiverSigRequired = true) and (account is sender)<br/>❌ - some failing scenarios described above |
 | beneficiary same as contract address | ✅ burns the eth and destructs the contract | ❌ - fails with SELF_DESTRUCT_TO_SELF                                                                                                                                                                       |
 
-### Call operations (CallOp, DelegateCallOp, CallCodeOp, StaticCallOp)
+### Call operations that do not transfer value (CallOp, DelegateCallOp, CallCodeOp, StaticCallOp)
+
+_Please note that the expected behavior above is valid considering there is no `value` passed. If there is `value`, this falls in the next section ("Transfer and send operations")._
```

Comment:
"... expected behavior **described in this section** is valid **if** there is no `value` passed" <- suggested rewording